main.go

```diff
@@ -129,3 +135,48 @@ func main() {
 	}
 	log.Infoln("Docker Model Runner stopped")
 }
+
+// createLlamaCppConfigFromEnv creates a LlamaCppConfig from environment variables
+func createLlamaCppConfigFromEnv() config.BackendConfig {
+	// Check if any configuration environment variables are set
+	argsStr := os.Getenv("LLAMA_ARGS")
+
+	// If no environment variables are set, use default configuration
+	if argsStr == "" {
+		return nil // nil will cause the backend to use its default configuration
+	}
+
+	// Split the string by spaces, respecting quoted arguments
+	args := splitArgs(argsStr)
+	log.Infof("Using custom arguments: %v", args)
+	return &llamacpp.Config{
```

Comment:
nit: Perhaps we should error out if `LLAMA_ARGS` tries to set args we need to control (`--model`, `--host`, `--embeddings`, and `--mmproj` in the future)